evap/staff/forms.py

```diff
@@ -49,10 +72,8 @@ def __init__(self, *args, **kwargs):
         self.fields['course'].choices = choices
 
     def clean(self):
-        if self.cleaned_data['course'] and self.cleaned_data['excel_file']:
-            raise ValidationError('Please select only one of course or Excel file.')
-        if not self.cleaned_data['course'] and not self.cleaned_data['excel_file']:
-            raise ValidationError('Please select either course or Excel File.')
+        if self.cleaned_data['course'] is None:
+            raise ValidationError("Please select a Course from the drop-down.")
```

Comment:
That would make the view's code larger again, so I think keeping it this way is better. I just found out that the unnecessary error message also exists in this commit, but i've fixed it now.